src/python/qepsi4/_psi4.py

```diff
@@ -2,44 +2,45 @@
 import numpy as np
 from openfermion import InteractionOperator, general_basis_change, MolecularData
 from openfermion.config import EQ_TOLERANCE
+from typing import Tuple
 
 
 def run_psi4(
-    geometry,
-    basis="STO-3G",
-    multiplicity=1,
-    charge=0,
-    method="scf",
-    reference="rhf",
-    freeze_core=False,
-    n_active=None,
-    save_hamiltonian=False,
-    options=None,
-    n_active_extract=None,
-    n_occupied_extract=None,
-    freeze_core_extract=False,
-):
+    geometry: dict,
+    basis: str = "STO-3G",
+    multiplicity: int = 1,
+    charge: int = 0,
+    method: str = "scf",
+    reference: str = "rhf",
+    freeze_core: bool = False,
+    n_active: int = None,
+    save_hamiltonian: bool = False,
+    options: dict = None,
+    n_active_extract: int = None,
+    n_occupied_extract: int = None,
+    freeze_core_extract: bool = False,
+) -> Tuple[dict, InteractionOperator]:
     """Generate an input file in the Psi4 python domain-specific language for
     a molecule.
     
     Args:
-        geometry (dict): a dictionary containing the molecule geometry.
-        basis (str): which basis set to use
-        multiplicity (int): spin multiplicity
-        charge (int): charge of the molecule
-        method (str): which calculation method to use
-        reference (str): which reference wavefunction to use. fno- energy methods are only compatible with RHF
-        freeze_core (bool): Whether to freeze occupied core orbitals
-        n_active (int): Number of active orbitals. Freeze virtual orbitals that do not fit on the qubits.
-        save_hamiltonian (bool): whether to save the Hamiltonian to a file. If True, symmetry will be disabled.
-        options (dict): additional commands to be passed to Psi4
-        n_active_extract (int): number of molecular orbitals to include in the
+        geometry: a dictionary containing the molecule geometry.
+        basis: which basis set to use
+        multiplicity: spin multiplicity
+        charge: charge of the molecule
+        method: which calculation method to use
+        reference: which reference wavefunction to use. fno- energy methods are only compatible with RHF
+        freeze_core: Whether to freeze occupied core orbitals
+        n_active: Number of active orbitals. Freeze virtual orbitals that do not fit on the qubits.
```

Comment:
Maybe it is not relevant for this particular PR but is `n_active` used anywhere in the code? (except for that single line where we set C1 symmetry but I guess that's not its original purpose)

Reply:
It doesn't seemed to be used other than that. It's probably a zmachine feature that wasn't fully migrated... I'll add a story about it.

Reply:
Or actually do you want to just remove it when you refactor the active space options?

Reply:
Sure! I've already removed it in my branch :)